samples/kubeflow/README.md

```diff
@@ -11,52 +12,91 @@ This demo uses the default stack:
 
 ## Installation
 
-Run the following script to install The Mesh for Data and all dependencies required for this demo. It will take a while for all containers to be ready due to image downloads.
+Run the following script to install The Mesh for Data core components and third party dependencies required for this demo.
+It will take a while for all containers to be ready due to image downloads. 
 
+```bash
+kubectl config set-context --current --namespace=m4d-system
+
+# For running on OpenShift
+WITHOUT_OPENSHIFT=false ./hack/install.sh
 
+# For running on Kind
+./hack/install.sh
+```
+
+Install kubeflow on your cluster. This sample (and code) was tasted against kubeflow v1.0.2.
+On a Kind cluster for example you can use the following script to install it:
 ```bash
-kind create cluster
-./install/install.sh
+cd samples/kubeflow/install/kubeflow
+./install_kubeflow.sh
+
+cd -
+```
+
+Install the Arrow-Flight module
+```
+kc apply -f https://raw.githubusercontent.com/IBM/the-mesh-for-data-flight-module/master/module.yaml
 ```
 
 ## Data owner instructions
 
-We already have transactions.csv stored in COS (TODO: don't assume that).
-Register the asset using the following command:
+Upload the file `data.csv` to an object-storage of your choice. For example, we tested the code against IBM's COS.
+`data.csv` contains the first 100 rows from the following [data set](https://www.kaggle.com/ntnu-testimon/paysim1/data)
+
+After uploading the data to some object-storage, alter the `fullPath` field in the json `example_transactions.csv.json` to contain the details of your object storage location, etc.
+See the comments in [third_pary/egeria/usage/create_new_asset.sh](../../third_party/egeria/usage/create_new_asset.sh) for more details.
+
+Then, register the asset using the following command:
 
 ```bash
+
+cd third_party/egeria/usage
+
+# Wait for the port-forward to take effect before proceeding to the next command
 kubectl port-forward -n egeria-catalog svc/lab-core 9443:9443 &
-sleep 5
-cd dataowner
-./create_new_asset.sh transactions.csv.json 'finance'
+
+
+# path-to-transactions.csv.json can be for example ../../../samples/kubeflow/example_transactions.csv.json for using the example file
+./create_new_asset.sh <path-to-transactions.csv.json> 'finance'
+
 cd -
 kill $!
 ```
 
-Note the asset ID that is listed next to "Read asset info: ". You will need it later:
+Save the asset ID (example for asset-id is 5de27155-48d3-4d78-8767-73e7b264e394).
+Export it to environment variable:
 ```
 export ASSET_ID=<asset-id>
 ```
 
-The credentials for the asset need to be registered in Vault. 
-You can do that using the Vault UI. First use this command to be able to open the UI from the browser:
+The credentials for the asset need to be registered in Vault. Currently, we're supporting hmac credentials only.
```

Comment:
Currently, we're supporting hmac credentials only.
--->
Currently, Only  hmac credentials are supported.

Reply:
Fixed